src/figuresClasses.ts

```diff
@@ -1,11 +1,103 @@
-export interface Figure {}
+export interface Figure {
+  shape: 'triangle' | 'rectangle' | 'circle';
+  color: 'red' | 'green' | 'blue';
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  a: number;
+
+  b: number;
+
+  c: number;
+
+  constructor(
+    a: number,
+    b: number,
+    c: number,
+    public shape: 'triangle',
+    public color: 'red' | 'green' | 'blue',
+  ) {
+    this.a = a;
+    this.b = b;
+    this.c = c;
+
+    if (this.a <= 0 || this.b <= 0 || this.c <= 0) {
+      throw new Error('Length is not allowed to be lower than zero');
```

Comment:
[CODE STYLE] - The error message thrown here could be more descriptive. Instead of 'Length is not allowed to be lower than zero', consider specifying which length (side) is problematic, and that it must be greater than zero.